components/automate-cli/cmd/chef-automate/migration_pg.go

```diff
@@ -201,6 +208,27 @@ func runCleanup(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func checkSpaceAvailable(dataDir string) (bool, error) {
+	os_path := getHabRootPath(habrootcmd)
```

Comment:
Use camelCase instead of underscore for variable name.

Reply:
Done